src/lumigo_tracer/spans_container.py

```diff
@@ -265,7 +266,7 @@ def create_span(cls, event=None, context=None, force=False) -> "SpansContainer":
         if Configuration.verbose:
             additional_info.update(
                 {
-                    "event": prepare_large_data(omit_keys(event)),
+                    "event": prepare_large_data(omit_keys(parse_event(event))),
```

Comment:
why is it here?
I think that it better be in the `parser` file (which is the file that parses the HTTP request and response).
The infra there already support the structure of finding the "right" praser, and parse the data.